File: pset6/mario/less/mario.py
import cs50

# If the height is >= 1 or n <= 8 means between 1 and 8 included, break the loop
while True:
    # Get the height from the user
    n = cs50.get_int("Height: ")
    if n >= 1 and n <= 8:
        break

# Iterate from 1 through the height int
# Iteration i, print i hashes and then a newline    
for i in range(n):
    # Iteration j, print j hashes and i spaces and then a newline
    for j in range(n):
        if i + j >= n - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()

File: Problem Set 01/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int n;

    do
    {
        // Get the height from the user
        n = get_int("Height: ");
    }
    // If the height is < 1 or > 8 or not an int, go back to step one
    while (n > 8 || n < 1);

    // Iterate from 1 through the height int
    // Iteration i, print i hashes and then a newline
    for (int i = 0; i < n; i++)
    {
        // Iteration j, print j hashes and i spaces and then a newline
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }

}